### PR TITLE
Small Readme suggestions

### PR DESCRIPTION
**Pull request by hartator**
Excellent work @zyc9012. Thank you again! 💯 

This PR is just a couple of small suggestions to improve the Readme. Mostly clarifying Install instructions, a little more aggressiveness in the marketing of the introduction, and mirroring a little more Nokogiri's own Readme quickstart.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,11 +2,37 @@
 
 [![CI](https://github.com/serpapi/nokolexbor/actions/workflows/ci.yml/badge.svg)](https://github.com/serpapi/nokolexbor/actions/workflows/ci.yml)
 
-A high performance HTML5 parser for Ruby based on [Lexbor](https://github.com/lexbor/lexbor/), with support for both CSS selectors and XPath. It's API is designed to be compatible with [Nokogiri](https://github.com/sparklemotion/nokogiri).
+Nokolexbor is a drop-in replacement for Nokogiri. It's 5.2x faster at parsing HTML and up to 997x faster at CSS selectors.
+
+It's a performance-focused HTML/XML parser for Ruby based on [Lexbor](https://github.com/lexbor/lexbor/). It supports both CSS selectors and XPath. Nokolexbor's API is designed to be 1:1 compatible as much as possible with [Nokogiri's API](https://github.com/sparklemotion/nokogiri).
```
Comment (zyc9012) [inline README.md:7]:
```suggestion
It's a performance-focused HTML parser for Ruby based on [Lexbor](https://github.com/lexbor/lexbor/). It supports both CSS selectors and XPath. Nokolexbor's API is designed to be 1:1 compatible as much as possible with [Nokogiri's API](https://github.com/sparklemotion/nokogiri).
```

**Review by zyc9012 — CHANGES_REQUESTED** [review]
Thanks @hartator 👍, just some minor changes.

It's only for HTML parsing, does not handle general XML well at the moment.

Let's also update the example selectors to fit the contents of `URI.open('https://github.com/serpapi/nokolexbor')`

```
# Search for nodes by css
doc.css('#readme h1', 'article h2', 'p[dir=auto]').each do |link|
  puts link.content
end

# Search for text nodes by css
doc.css('#readme p > ::text').each do |text|
  puts text.content
end

# Search for nodes by xpath
doc.xpath('//div[@id="readme"]//h1', '//article//h2').each do |link|
  puts link.content
end
```

**Comment by hartator** [comment]
All sounds good @zyc9012. I've just made the updates. 👍

**Review by zyc9012 — APPROVED** [review]
All looks good, thank you @hartator 👍

**Pull request merged**